### DISCUSSION TITLE
Installation via Softaculous - ver 1.30 getting 503 service unavailable

### DISCUSSION THREAD

**Discussion by belacs2021** [kimai/kimai#3790] [On-Premise: Installation & Updates]
Quick question - i am having lots of Service unavailable error once using the application which was installed via cpanel softaculous. Is it possible to upgrade to version 2.0 which seems to be the recommended option as long as I have shell access. I don't see any update option in the present installation. Please advise - thank you. note I am only low level knowledgeable on coding/programming. Thank you

**Comment by kevinpapst** [comment]
Please ask at Softaculous how to upgrade. 
I have no clue how they install Kimai and why you suffer from 503 errors. 
But regular Kimai installations do not have that.
